jsonschema-hyperschema.xml

```diff
@@ -2029,104 +2093,72 @@ Link: <https://api.example.com/trees/1/nodes/456> rev=up
+                        then the context URIs would contain fragments identical to
+                        the context pointer field.  For application/json and other media types
+                        without fragments, it is critically important to consider the context
+                        pointer as well as the context URI.
                     </postamble>
                 </figure>
                 <t>
-                    But what if we do not have any "thing"s yet?  We cannot get to the
-                    individual "thing" schema as there is no individual "thing" to fetch.
-                    And the "tag:rel.example.com,2017:thing" link in the entry point
-                    resource does not indicate how to create a "thing".  The "self" link
-                    requires the "id" to already exist, but it is assigned by the server.
-                    So we need to add another link to our entry point schema:
+                    There are three "self" links, one for the collection, and one for
+                    each item in the "elements" array.  The item "self" links are defined
+                    in the individual "thing" schema which is referenced with "$ref".
+                    The three links can be distinguished by their context or attachment
+                    pointers.  We will revisit the "submissionSchema" of the collection's
+                    "self" link in <xref target="firstItem"/>.
                 </t>
-                <figure>
-                    <preamble>
-                        This LDO would be added to the top-level "links" array in the entry
-                        point resource's hyper-schema.
-                    </preamble>
-                    <artwork>
-<![CDATA[{
-    "rel": "tag:rel.example.com,2017:thing-collection",
-    "href": "/things",
-    "submissionSchema": {
-        "$ref": "thing#"
-    },
-    "targetSchema": {
-        "$ref": "thing-collection#"
-    }
-}]]>
-                    </artwork>
-                </figure>
                 <t>
-                    <cref>
-                        Here we also see the "submissionSchema" to use to create a "thing",
-                        but how do we recognize it?  We can't use a "collection" link relation
-                        here, because there is no identifiable "thing" to serve as the context
-                        resource.  We could look at the submission schema, notice that it
-                        has a "collection" link and is therefore an item, and notice that
-                        its "collection" link produces the same(-ish?) URI, but that seems
-                        overly complicated.  And gets more so once pagination and filtering
-                        are introduced.
-                    </cref>
+                    There are two "item" links, one for each item in the "elements" array.
+                    Unilke the "self" links, these are defined only in the collection schema.
```

Comment:
I swear I ran the spell checker!  Repeatedly!  Argh... I need to find an intelligent spell-checker for XML in vim that I can just leave on instead of the convoluted filetype switching that I do to get the regular one to work at all.